.github/bin/common.sh

```diff
@@ -167,24 +167,49 @@ installYq() {
 # Graps Eclipse Che logs
 collectLogs() {
   mkdir -p ${ARTIFACTS_DIR}
-  chectl server:logs --chenamespace=${NAMESPACE} --directory=${ARTIFACTS_DIR}
 
-  set +x
+  set +e
+  chectl server:logs --chenamespace=${NAMESPACE} --directory=${ARTIFACTS_DIR}
+  collectDevworkspaceOperatorLogs
   oc get events -n ${DEVWORKSPACE_CONTROLLER_TEST_NAMESPACE} > ${ARTIFACTS_DIR}/events-${DEVWORKSPACE_CONTROLLER_TEST_NAMESPACE}.txt
   oc get events -n ${DEVWORKSPACE_CHE_OPERATOR_TEST_NAMESPACE} > ${ARTIFACTS_DIR}/events-${DEVWORKSPACE_CHE_OPERATOR_TEST_NAMESPACE}.txt
-  set -x
+  set -e
+}
+
+collectDevworkspaceOperatorLogs() {
+  mkdir -p ${ARTIFACTS_DIR}/devworkspace-operator
+
+  oc get events -n devworkspace-controller > ${ARTIFACTS_DIR}/events-devworkspace-controller.txt
+
+  #determine the name of the devworkspace controller manager pod
+  local CONTROLLER_POD_NAME=$(oc get pods -n devworkspace-controller -l app.kubernetes.io/name=devworkspace-controller -o json | jq -r '.items[0].metadata.name')
+  local WEBHOOK_SVR_POD_NAME=$(oc get pods -n devworkspace-controller -l app.kubernetes.io/name=devworkspace-webhook-server -o json | jq -r '.items[0].metadata.name')
+  
+  # save the logs of all the containers in the DWO pod
+  for container in $(oc get pod -n devworkspace-controller ${CONTROLLER_POD_NAME} -o json | jq -r '.spec.containers[] | .name'); do
+    mkdir -p ${ARTIFACTS_DIR}/devworkspace-operator/${CONTROLLER_POD_NAME}
```

Comment:
What if we just bump everything in the namespace? Like we do on dashboard/devworkspace operator side https://github.com/devfile/devworkspace-operator/blob/main/.ci/common.sh#L18.
You just need to specify namespace as an argument.

Also, please note that chectl:logs extract only certain pods, (not sure if che-gateway is respected), so in additional to chectl:logs, we also do bummPodsInfo eclipse-che, and we have all jobs, all containers info.

Reply:
Created https://github.com/eclipse/che/issues/20504 for that.